cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -176,6 +176,17 @@
         }
     }
 
+    .respond-to-max( @bp-xs-max, {
```

Comment:
While you're in here, could you add a code comment above this for `// Mobile size` for consistency with other code comments further down?
I know we have breakpoints that don't follow devices, but effectively the menu has device sizes.